tests/install_testbed/generate_region_config.py

```diff
@@ -0,0 +1,136 @@
+#!/usr/bin/env python
+# -- encoding: utf-8 --
+#
+# Copyright 2015 Telefónica Investigación y Desarrollo, S.A.U
+#
+# This file is part of FI-Core project.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+#
+# You may obtain a copy of the License at:
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# For those usages not covered by the Apache version 2.0 License please
+# contact with opensource@tid.es
+#
+__author__ = 'chema'
+
+from subprocess import Popen, PIPE
+import os
+
+# default content of the properties file. The passwords are
+# also append to this file.
+
+default_values = {
+    'KEYSTONE_HOST': '127.0.0.1',
+    'REGION': 'RegionOne',
+    'CONTROLLER': '127.0.0.1',
+    'PUBLIC_CONTROLLER': '127.0.0.1',
+    'EXTERNAL_INTERFACE': 'eth2',
+    'MANAGEMENT_IPS': '127.0.0',
+    'TUNNEL_IPS': '192.168.57',
+}
+
+template = """
```

Comment:
mmm, I would suggest to put it into a text file with only real permissions the get it from the file. It is a long text to put inside the python code.